lib/handlers_v5.js

```diff
@@ -341,3 +344,21 @@ function shutdown_request(request) {
 
     status_idle.call(this, request);
 }
+
+function is_complete_request(request) {
```

Comment:
`jp-kernel` is used by the coffeescript and babel kernels.

For these kernels, we could do something like this:

```js
var code = request.content.code;

if (this.session.transpile) {
    try {
        code = this.session.transpile(code);
    } catch (err) {
        request.respond(this.shellSocket, "is_complete_reply", {status: "unknown"});
        return;
    }
}

// the code below should use `code` instead of `request.content.code` 
```